src/Axe.Windows.Desktop/Keyboard/HotkeysHandler.cs

```diff
@@ -112,6 +112,9 @@ protected virtual void Dispose(bool disposing)
                     }
 
                     this.HotKeyList.Clear();
+
+                    source?.Dispose();
```

Comment:
>source [](start = 20, length = 6)

This is a behavioral change, but it seems totally safe. Happy to suppress the warning if that feels "safer"